app/instrumentation-tests/src/org/commcare/androidTests/AdvancedSettingsTests.kt

```diff
@@ -0,0 +1,212 @@
+
+        assertTrue(onView(withText("1 form sent to server!")).isPresent())
+
+        InstrumentationUtility.openGridForm(0,1)
+        onView(withId(R.id.search_action_bar)).perform(click())
+        onView(withId(R.id.search_src_text)).perform(typeText(NAME))
+        InstrumentationUtility.clickListItem(R.id.screen_entity_select_list, 0)
+        onView(isRoot()).perform(ViewActions.swipeLeft())
+        assertFalse(onView(withText("FINISH")).isPresent())
+        onView(withText("CONTINUE")).perform(click())
+        assertTrue(onView(withText("FINISH")).isPresent())
+        onView(withText("Boston")).perform(click())
+        InstrumentationUtility.submitForm()
+
+        InstrumentationUtility.openGridForm(0,1)
+        onView(withId(R.id.search_action_bar)).perform(click())
+        onView(withId(R.id.search_src_text)).perform(typeText(NAME))
+        val formatter = DateTimeFormatter.ofPattern("yy/MM/d")
+        var timeNow = (LocalDateTime.now()).format(formatter)
+        val text1 = InstrumentationUtility.getText(onView(withSubstring(timeNow)))
+        Log.i("time1:", text1)
+        Thread.sleep(15000)
+        timeNow = (LocalDateTime.now()).format(formatter)
+        val text2 = InstrumentationUtility.getText(onView(withSubstring(timeNow)))
+        Log.i("time2:", text2)
+        assertNotEquals(text1, text2)
+        InstrumentationUtility.gotoHome()
+
+    }
+
+    private fun setPin(pin: String){
+        assertTrue(onView(withText("Enter a 4-digit PIN")).isPresent())
+        onView(withClassName(Matchers.endsWith("EditText")))
+            .perform(ViewActions.typeText((pin)))
+        onView(withText("CONTINUE")).perform(click())
+        assertTrue(onView(withText("Re-Enter the PIN you chose")).isPresent())
+        onView(withClassName(Matchers.endsWith("EditText")))
+            .perform(ViewActions.typeText((pin)))
+        onView(withText("CONFIRM")).perform(click())
+        InstrumentationUtility.waitForView(withText("PIN set successfully"))
+    }
+    private fun verifyCreatePINWindow(){
+        assertTrue(onView(withText("YES, SET MY PIN NOW")).isPresent())
+        assertTrue(onView(withText("NO, BUT ASK AGAIN ON NEXT LOGIN")).isPresent())
+        assertTrue(onView(withText("NO, AND DON'T ASK AGAIN")).isPresent())
+        assertTrue(onView(withId(R.id.extra_info_button)).isPresent())
+    }
+
+    private fun reportIssue(reportText: String){
+        InstrumentationUtility.enterText(R.id.ReportText01,reportText)
+        onView(ViewMatchers.isRoot()).perform(ViewActions.closeSoftKeyboard())
+        onView(withText("SUBMIT REPORT")).isPresent()
+
+        var intentcallback = IntentCallback { intent ->
+            var extraText = intent.extras!!.getString(Intent.EXTRA_TEXT)
+            TestCase.assertTrue(extraText!!.contains(reportText))
+        }
+        IntentMonitorRegistry.getInstance().addIntentCallback(intentcallback)
+        onView(withText("SUBMIT REPORT")).perform(click())
```

Comment:
we should stub the intent before clicking submit as we don't want it to send fake issues on Support team